src/QsCompiler/Tests.Compiler/CallGraphTests.fs

```diff
@@ -76,19 +57,14 @@ type CallGraphTests (output:ITestOutputHelper) =
         let srcChunks = ReadAndChunkSourceFile fileName
         srcChunks.Length >= testNumber |> Assert.True
         let compilationDataStructures = BuildContent srcChunks.[testNumber-1]
-        let callGraph = BuildCallGraph.Apply compilationDataStructures.BuiltCompilation
-        Assert.NotNull callGraph
-        callGraph
-
-    let CompileCycleDetectionTest testNumber =
-        let callGraph = CompileTest testNumber "CycleDetection.qs"
-        callGraph.GetCallCycles ()
+        Assert.NotNull compilationDataStructures.BuiltCompilation
+        compilationDataStructures.BuiltCompilation
 
-    let CompileCycleValidationTest testNumber =
-        let graph = CompileTest testNumber "CycleValidation.qs"
+    let BuildTrimmedGraph (compilation : QsCompilation) = SimpleCallGraph(compilation, true)
 
-        graph.GetCallCycles()
-        |> Seq.map (fun x -> GetEdges graph x) |> Seq.collect (fun x -> CartesianProduct x)
+    let CompileCycleDetectionTest testNumber =
+        let SimpleCallGraph = CompileTest testNumber "CycleDetection.qs" |> SimpleCallGraph
```

Comment:
Are there also tests for `ConcreteCallGraph`? I didn't see them.

Reply:
There are no tests specifically for concrete call graph. There will be a PR similar to #621 for the use case of the concrete call graph in monomorphization that will test the concrete call graph.

Reply:
Could you add a TODO comment for concrete call graph tests?

Reply:
I'm not at this time foreseeing any tests specific to the concrete call graph. All the tests for the call graph work in general for call graphs, and the tests for monomorphization will be sufficient to test the specifics of the concrete graph.

Reply:
It's probably OK if it's only tested via monomorphization, but a TODO mentioning that the concrete call graph is currently not tested until then would be nice.

Reply:
The monomorphization tests I believe don't current capture type argument information, do they? In any case, a separate set of tests for the additional information in the concrete call graph is warranted here.

Reply:
I've added a ToDo for the concrete call graph tests. I don't want to add tests for it in this PR to keep it from getting any bigger, but I will look into some more specific tests for it in the PR that is specific to the concrete call graph.